components/announcement_bar/version_bar/version_bar.tsx

```diff
@@ -11,20 +10,24 @@ import {AnnouncementBarTypes} from 'utils/constants';
 
 import AnnouncementBar from '../default_announcement_bar';
 
-export default class VersionBar extends React.PureComponent {
-    static propTypes = {
-        serverVersion: PropTypes.string,
-    };
+interface IMyComponentProps {
```

Comment:
Usually we name these just `Props` and `State` if it's not exported, there is no need to be specific.